Chummer/Classes/clsOptions.cs

```diff
@@ -45,6 +45,15 @@ public enum ClipboardContentType
         Lifestyle,
     }
 
+    public enum EnumUseLoggingApplicationInsights
```

Comment:
We got intellisense, we don't need the name to also tell us its an enum. There is a lot of old code in chummer that does this, and a lot of old programmers that does this, but unless you know something than the typechecker don't, it is only extra text I have to read.